app/models/post.rb

```diff
@@ -0,0 +1,10 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments,
```

Comment:
```suggestion
  belongs_to :author, class_name: 'User'
  has_many :comments,
  has_many :likes
  
```

- [ ] Kindly update your post model to have many likes 👍🏿

Reply:
This change has been implemented well